test/test/jfrconverter/JfrconverterTests.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.jfrconverter;
+
+import one.convert.*;
+import one.profiler.test.*;
+
+// Simple smoke tests for JFR converter. The output is not inspected for errors,
+// we only verify that the conversion completes successfully.
+public class JfrconverterTests {
+
+    @Test(mainClass = CpuBurner.class, agentArgs = "start,jfr,file=profile.jfr,all,file=%f")
+    public void allocationHeatmapConversion(TestProcess p) throws Exception {
+        p.waitForExit();
+        assert p.exitCode() == 0;
+        JfrToHeatmap.convert("profile.jfr", "/dev/null", new Arguments("--alloc"));
+    }
+
+    @Test(sh = "")
+    public void cpuHeatmapConversion(TestProcess p) throws Exception {
+        JfrToHeatmap.convert("profile.jfr", "/dev/null", new Arguments("--cpu"));
```

Comment:
Looks like this has dependency to an artifact generated in a different test. Can we make the tests independent from each other and runnable in any order?

Reply:
You can generate the jfr for each test separately.

Reply:
I'm particularly against producing the same JFR 3 times in a row just to run converter with slightly different options. This is suboptimal wrt. test run time and resource usage.

Reply:
I'm with @krk here, having three conceptually different tests doing very different things all in one test looks confusing

Reply:
> conceptually different tests doing very different things

I don't fully agree. These tests run exactly the same program with exactly the same arguments. The only difference is parameters of conversion. OK, maybe FlameGraph conversion and Heatmap conversion can be split into separate tests, but rerunning the test program from scratch and reprofiling it again with the same parameters just to change one heatmap option does not make much sense to me. It's a waste of time and compute resources. Coverage will not get any bigger.

Reply:
Alright, we can have a meaningful compromise. What about making two tests: one for Heatmap conversion and another one for FlameGraph conversion? This will allow keeping the test run time short and the amount of test code small, while increasing code coverage by adding more conversions with different arguments.

Reply:
3b433e5555671a25a14f20a973943666bb6137c4